.build/build-resolver.xml

```diff
@@ -290,13 +290,12 @@
             <url url="${lib.download.base.url}/lib/sigar-bin/libsigar-x86-solaris.so"/>
         </get>
 
-        <!-- Amazon Corretto Crypto Providers for various architectures -->
-        <resolve>
-            <dependencies>
-                <dependency groupId="software.amazon.cryptools" artifactId="AmazonCorrettoCryptoProvider" version="2.2.0" classifier="linux-x86_64" />
-                <dependency groupId="software.amazon.cryptools" artifactId="AmazonCorrettoCryptoProvider" version="2.2.0" classifier="linux-aarch_64" />
-            </dependencies>
-        </resolve>
+        <get src="${artifact.github.release}/corretto/amazon-corretto-crypto-provider/releases/download/2.2.0/AmazonCorrettoCryptoProvider-2.2.0-linux-x86_64.jar"
+             dest="${local.repository}/software/amazon/cryptools/AmazonCorrettoCryptoProvider/2.2.0/AmazonCorrettoCryptoProvider-2.2.0-linux-x86_64.jar"
+             usetimestamp="true" quiet="true" skipexisting="true"/>
+        <get src="${artifact.github.release}/corretto/amazon-corretto-crypto-provider/releases/download/2.2.0/AmazonCorrettoCryptoProvider-2.2.0-linux-aarch_64.jar"
+             dest="${local.repository}/software/amazon/cryptools/AmazonCorrettoCryptoProvider/2.2.0/AmazonCorrettoCryptoProvider-2.2.0-linux-aarch_64.jar"
+             usetimestamp="true" quiet="true" skipexisting="true"/>
```

Comment:
why was this re-introduced? 

i'm also confused as to why the macro needed to be moved out… `resolver-init` defines a number of macros that are used in other places…

Reply:
@michaelsembwever it didnt work :( I noticed that while doing build in circle. You can have another shot at that ... it just ... didnt donwload anything.

Reply:
@michaelsembwever 

I am getting this

_resolver-dist-lib_get_files:
    [retry] Attempt [0]: error occurred; retrying...
    [retry] Attempt [1]: error occurred; retrying...
    [retry] Attempt [2]: error occurred; retrying...
    [retry] Attempt [0]: error occurred; retrying after 10 ms...

_resolver-dist-lib_get_files:
    [retry] Attempt [0]: error occurred; retrying...
    [retry] Attempt [1]: error occurred; retrying...
    [retry] Attempt [2]: error occurred; retrying...
    [retry] Attempt [1]: error occurred; retrying after 10 ms...

_resolver-dist-lib_get_files:
    [retry] Attempt [0]: error occurred; retrying...
    [retry] Attempt [1]: error occurred; retrying...
    [retry] Attempt [2]: error occurred; retrying...
    [retry] Attempt [2]: error occurred; retrying after 10 ms...

_resolver-dist-lib_get_files:
    [retry] Attempt [0]: error occurred; retrying...
    [retry] Attempt [1]: error occurred; retrying...
    [retry] Attempt [2]: error occurred; retrying...

Reply:
i'll take a look

Reply:
cherry-pick this bad boy
https://github.com/thelastpickle/cassandra/commit/58f5a9c04a3c32e5da5298de3638424614138f38

Reply:
That helped! Thank you.